cf-agent/Makefile.am

```diff
@@ -124,7 +124,11 @@ if !BUILTIN_EXTENSIONS
   bin_PROGRAMS = cf-agent
 
   # Workaround for automake madness (try removing it if you want to know why).
-  cf_agent_CFLAGS = $(AM_CFLAGS)
+if FREEBSD
+    cf_agent_CFLAGS = $(AM_CFLAGS) -lutil
```

Comment:
`-lutil` should definitely go to `CFLAGS`, it's a linker flag. So I guess this should actually go to `AM_LDFLAGS` above?

Reply:
fixed